GlobalDataverseCommunityConsortium/dataverse-ansible
# Added a molecule test.

The PR adds a [molecule](https://molecule.readthedocs.io/en/latest/) test. It is work in progress, but I would appreciate any feedback. Currently, I am unable to deploy Dataverse in Payara, so I am using the old Glassfish 4.1.

To test:

1. Install [Vagrant](https://www.vagrantup.com/), [VirtualBox](https://www.virtualbox.org/).
2. Install molecule and the vagrant driver:

        pip install molecule molecule-vagrant

3. Execute: create, converge and verify:
 
        molecule create
        molecule converge
        molecule verify

`molecule test` will fail at the idempotence check, as the role is not yet idempotent.
